### ISSUE TITLE
Primary / Secondary contact nomination

### ISSUE THREAD

**Issue by ste78** [alces-software/alces-flight-center#303]
Allow a primary or secondary contact to give their position to another view-only contact on the site. Thereby making them a view only contact and the nominated person a primary/secondary contact able to log cases.

**Comment by bobwhitelock** [comment]
Trello: https://trello.com/c/4gBdmLRo/336-primary-secondary-contact-nomination

**Comment by bobwhitelock** [comment]
@ste78 you were saying yesterday you would prefer us to do the MVP version of each of these features first - for this the most minimal way I can think of to meet this requirement is:

1. we implement https://github.com/alces-software/alces-flight-center/issues/301;

2. we add a new 'Switch my contact position' Issue to our Issue hierarchy, which has a level 2 Tier which:

    a. documents what it means to do this;

    b. has a field where the name of a view-only user to switch your contact position with can be entered;

3. when we receive a Case for this Issue, we switch the requesting user to be view-only and the requested user to be a site contact (in the admin interface), and then inform them that we have done this as for any other Case (resolve the Case and comment).

I believe this would require just data changes, and no additional code changes over https://github.com/alces-software/alces-flight-center/issues/301. Would this be a sufficient MVP version?

Later, if needed, we could make this a Tier 1 Case which automatically performs the switch when applied.

**Comment by ste78** [comment]
yes @bobwhitelock I was and yes that sounds amazing and exactly what we need for a MVP 👍 👍 👍

**Comment by bobwhitelock** [comment]
@ste78 this new issue doesn't seem to fit neatly into our existing issue hierarchy, so I was planning to classify it like this: HPC Environment (existing service) > Flight Center management (new category) > Nominate new site contact (new issue). Let me know if you want it to work any differently :slightly_smiling_face:.

**Comment by ste78** [comment]
sgtm!
